### PR TITLE
my hw jv logger

### PR DESCRIPTION
**Pull request by VolodymyrKachan**

### CODE REVIEW

File: src/main/java/mate/academy/service/AuthenticationServiceImpl.java [hunk 1]
```diff
@@ -2,11 +2,16 @@
...
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
         //TODO: add corresponding log message about method login was called
```
Comment (fliarkovskyi) [inline src/main/java/mate/academy/service/AuthenticationServiceImpl.java:13]: image

File: src/main/java/mate/academy/service/OrderServiceImpl.java [hunk 2]
```diff
@@ -4,11 +4,16 @@
...
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
         // TODO: add log message about method completeOrder was called
```
Comment (fliarkovskyi) [inline src/main/java/mate/academy/service/OrderServiceImpl.java:15]: image

**Review by fliarkovskyi — APPROVED** [review]

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
+*.log
```
Comment (Noct2000) [inline .gitignore:3]: check your `log4j2.xml`
```suggestion
logs/*
```

File: src/main/resources/log4j2.xml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+        <Logger name="mate.academy" level="info" additivity="false">
+            <AppenderRef ref="LogToFile"/>
+            <AppenderRef ref="LogToConsole"/>
+        </Logger>
+        <Root level="error">
+            <AppenderRef ref="LogToFile" />
+            <AppenderRef ref="LogToConsole"/>
+        </Root>
+    </Loggers>
+</Configuration>
```
Comment (Noct2000) [inline src/main/resources/log4j2.xml:23]: add new line to the end of the file

**Review by Noct2000 — CHANGES_REQUESTED** [review]

**Review by Noct2000 — APPROVED** [review]

**Pull request closed without merging**